### PR TITLE
PrettyBytes

### PR DESCRIPTION
**Pull request by thomasdax98**
Automatically determines the fitting unit for a byte value and formats the number accordingly.

Supported units are: `"byte" | "kilobyte" | "megabyte" | "gigabyte" | "terabyte" | "petabyte"` (the same as react-intl)

Closes https://github.com/vivid-planet/comet-admin/issues/398

### CODE REVIEW

File: packages/admin-stories/src/admin/helpers/PrettyBytes.tsx [hunk 1]
```diff
@@ -0,0 +1,101 @@
+import { PrettyBytes } from "@comet/admin";
+import { storiesOf } from "@storybook/react";
+import * as React from "react";
+
+const Story = () => {
```
Comment (johnnyomair) [inline packages/admin-stories/src/admin/helpers/PrettyBytes.tsx:5]: This story and the documentation are identical. @manuelblum is there a way to use a story for both development and the documentation?

**Review by johnnyomair — APPROVED** [review]

File: packages/admin-stories/src/admin/helpers/PrettyBytes.tsx (hunk 1 above)
Comment (thomasdax98) [reply packages/admin-stories/src/admin/helpers/PrettyBytes.tsx:5]: Theoretically, there is a way (https://storybook.js.org/docs/react/api/mdx#embedding-stories). But I've never managed to make it work correctly. The story is displayed, however it's not possible to use the "Show Code" feature when embedding it like this. It always says "No Code".

Comment (nsams) [reply packages/admin-stories/src/admin/helpers/PrettyBytes.tsx:5]: see https://github.com/vivid-planet/comet-admin/pull/411 and talk to @manuelblum

Comment (nsams) [reply packages/admin-stories/src/admin/helpers/PrettyBytes.tsx:5]: I think it actually can make sense to have two (not identical) stories:

- development story: tests all possible combinations (like you currently do)
- doc stories: no reason to demonstrate the whole range from kB to PB, exactly one typical example (using for example MB) is enough.

Comment (thomasdax98) [reply packages/admin-stories/src/admin/helpers/PrettyBytes.tsx]: I shortened the examples in the docs, but added a "Complete Example" section at the end where I embedded the development story. Is that okay?

Comment (nsams) [reply packages/admin-stories/src/admin/helpers/PrettyBytes.tsx]: please remove the development story from the docs :)

the plan is to move those into a development section, see https://github.com/vivid-planet/comet-admin/pull/411#issuecomment-854716187 for now it's ok to have them in a stories section.

Comment (thomasdax98) [reply packages/admin-stories/src/admin/helpers/PrettyBytes.tsx]: @nsams Done

**Comment by nsams** [comment]
one minor thing: meanwhile the docs structure changed, please adjust to it (don't place it into `@admin`)

**Comment by thomasdax98** [comment]
@nsams The structure should be okay now

**Review by nsams — APPROVED** [review]

**Pull request merged**